cfgmgr/buffer_check_headroom_mellanox.lua

```diff
@@ -60,7 +60,8 @@ if is_port_with_8lanes(lanes) then
     pipeline_latency = pipeline_latency * 2 - 1
     egress_mirror_size = egress_mirror_size * 2
 end
-accumulative_size = accumulative_size + 2 * pipeline_latency * 1024 + egress_mirror_size
+local lossy_pg_size = pipeline_latency * 1024
```

Comment:
Why we no longer take "2* pipeline latency"?

Reply:
`2*` actually stood for 1 fixed overhead (which equals lossy PG overhead) and 1 lossy PG overhead.
Originally, we assumed there is only one lossy PG, and did not check lossy PG [(L99)](https://github.com/sonic-net/sonic-swss/pull/2763/files/c36bd2a45922cc4795f46f926fcdb144eac3da11#diff-674791033ce2618073ce2c199abc3d140620b0f10c1df96f5fbd353db3942afbL99), but just counted the lossy PG overhead into `accumulative_size` in line 63. But that is not accurate. A user can configure additional lossy PG in CONFIG_DB.
So, we only count `1 fixed overhead` into `accumulative_size` and count overhead for each lossy PG configured in CONFIG_DB.